fastdupes.py

```diff
@@ -127,7 +128,7 @@ def hashFile(handle, want_hex=False, limit=None, chunk_size=CHUNK_SIZE):
     for block in iter(lambda: handle.read(chunk_size), b''):
         fhash.update(block)
         read += chunk_size
-        if 0 < limit <= read:
+        if limit and limit <= read:
```

Comment:
Would you mind explaining why you made this no longer ensure that `limit` is positive?